framework/src/userobjects/MessageFromInput.C

```diff
@@ -0,0 +1,53 @@
+//* This file is part of the MOOSE framework
+//* https://www.mooseframework.org
+//*
+//* All rights reserved, see COPYRIGHT for full restrictions
+//* https://github.com/idaholab/moose/blob/master/COPYRIGHT
+//*
+//* Licensed under LGPL 2.1, please see LICENSE for details
+//* https://www.gnu.org/licenses/lgpl-2.1.html
+
+#include "MessageFromInput.h"
+#include "Console.h"
```

Comment:
```suggestion
#include "ConsoleStream.h"
```
I suspect this is the better one?